### PR TITLE
MAR-4613: Upgrade to Python 3

### PR DESCRIPTION
**Pull request by resh-ie**

### CODE REVIEW

File: requirements_for_test.txt [hunk 1]
```diff
@@ -1,76 +1,75 @@
...
 bcrypt==3.1.4
 blinker==1.4
 boto==2.49.0
 boto3==1.9.19
 botocore==1.12.19
 certifi==2018.8.24
-cffi==1.11.4
+cffi==1.14.5
 chardet==3.0.4
 Click==7.0
```
Comment (resh-ie) [inline requirements_for_test.txt]: updated this package to match the other repos

**Review by Revit — APPROVED** [review]

File: dmutils/email.py [hunk 2]
```diff
@@ -91,16 +87,16 @@ def send_email(to_email_addresses, email_body, subject, from_email, from_name, r
         return_address = current_app.config.get('DM_EMAIL_RETURN_ADDRESS')
 
         result = email_client.send_email(
-            Source=u"{} <{}>".format(from_name, from_email),
+            Source="{} <{}>".format(from_name, from_email),
             Destination=destination_addresses,
             Message={
                 'Subject': {
-                    'Data': subject,
+                    'Data': subject.decode(),
```
Comment (Revit) [inline dmutils/email.py:94]: No need to decode()

File: dmutils/email.py [hunk 3]
```diff
@@ -91,16 +87,16 @@ def send_email(to_email_addresses, email_body, subject, from_email, from_name, r
...
                     'Charset': 'UTF-8'
                 },
                 'Body': {
                     'Html': {
-                        'Data': email_body,
+                        'Data': email_body.decode(),
```
Comment (Revit) [inline dmutils/email.py:99]: No need to decode()

File: dmutils/email.py (hunk 2 above)
Comment (resh-ie) [reply dmutils/email.py:94]: @tobiasbrown should we remove [subject = to_bytes(subject](https://github.com/AusDTO/dto-digitalmarketplace-utils/pull/111/files/07fdee7c261c0770faf3ce5bfc4b7f9905247e88#diff-5c6407d010068da43fa9036bfd938dae7c9befeb2ea62fcee7c87fb21e7b3efeR71)) that way we don't need to add the decode function.

File: dmutils/email.py (hunk 3 above)
Comment (resh-ie) [reply dmutils/email.py:99]: @tobiasbrown should we remove `email_body = to_bytes(email_body)` that way we can remove `'Data': email_body.decode()`

File: dmutils/forms.py [hunk 4]
```diff
@@ -98,7 +98,7 @@ def csrf_context(self):
 
     def __init__(self, *args, **kwargs):
         if current_app.config['CSRF_ENABLED']:
-            self.Meta.csrf_secret = current_app.config['SECRET_KEY']
+            self.Meta.csrf_secret = current_app.config['SECRET_KEY'].encode()
```
Comment (Revit) [inline dmutils/forms.py:101]: not sure if we need encode() here

File: dmutils/email.py (hunk 2 above)
Comment (tobiasbrown) [reply dmutils/email.py:94]: Let's leave this as is.  Emails have been received during testing.

File: dmutils/email.py (hunk 3 above)
Comment (tobiasbrown) [reply dmutils/email.py:99]: Let's leave this as is. Emails have been received during testing.

File: dmutils/forms.py (hunk 4 above)
Comment (tobiasbrown) [reply dmutils/forms.py:101]: The csrf_secret needs to be a [bytes string](https://wtforms.readthedocs.io/en/2.3.x/csrf/#wtforms.csrf.session.SessionCSRF).

**Review by tobiasbrown — APPROVED** [review]

**Comment by resh-ie** [comment]
Changes looks good!

**Pull request merged**